### ISSUE TITLE
some questions about this experiments

### ISSUE THREAD

**Issue by yicocc** [amaralibey/gsv-cities#14]
![pdJwyWSF00](https://user-images.githubusercontent.com/51085010/226268714-19450a69-0ee9-4985-8103-03ba3d49302a.jpg)
In my device，this result in Nordland is like not good ,I think  maybe  is parameters is not correct,but I don't know how to adjust which parameters  .In addition,I want to ask for you  about the groundtruth of dataset how to make.Thank you!

**Comment by amaralibey** [comment]
Hello @yicocc,

Can you share more information about this issue?

- Did you download the Nordland dataset as explained? 
- Did you properly load the pre-trained weights? 
- Did you use the right augmentation (esp. normalization)?
